cli/src/pcluster/aws/cfn.py

```diff
@@ -31,13 +32,17 @@ def create_stack(
         self, stack_name: str, disable_rollback: bool, tags: list, template_body: str, parameters: list = None
     ):
         """Create CFN stack by using the given template."""
+        optional_args = {
```

Comment:
These lines are not covered by unit tests, see finding on the bottom of this page: https://github.com/aws/aws-parallelcluster/pull/5081/files